configs/components/rubygem-nokogiri.rb

```diff
@@ -7,9 +7,9 @@
     # :gem_home were then tar'ed up and packed into this container tarball.
     pkg.url "http://buildsources.delivery.puppetlabs.net/nokogiri-precompiled-huaweios-#{pkg.get_version}-for-ruby-#{settings[:ruby_version]}.tar.gz"
     if settings[:ruby_version] == "2.1.9"
-      pkg.md5sum "2ad8b4dd69f6e2b296dfaba293e9cb11"
-    elsif settings[:ruby_version] == "2.3.3"
-      pkg.md5sum "443e5b3bea092cd955e125c8e8bda0d6"
+      pkg.md5sum "4bf7a53bbfa5593041f7698f4e56a8d1"
+    elsif settings[:ruby_version] == "2.3.1"
+      pkg.md5sum "d11bcd650dc4942aa198586f4ff04247"
```

Comment:
Oh, we probably need to rebuild nokogiri as part of this mergeup - the CVE fork had more recent rebuilds, but we haven't rebuild to 2.3.3 for master. @ScottGarman what do you think needs doing here?